src/Orleans.PlatformServices/PlatformAssemblyLoader.cs

```diff
@@ -16,22 +12,7 @@ public static Assembly LoadFromBytes(byte[] assembly, byte[] debugSymbols = null
                 throw new ArgumentNullException(nameof(assembly));
             }
 
-#if NETCORE
-            using (var assemblyStream = new MemoryStream(assembly))
-            {
-                if (debugSymbols != null)
-                {
-                    using (var debugSymbolStream = new MemoryStream(debugSymbols))
-                    {
-                        return AssemblyLoadContext.Default.LoadFromStream(assemblyStream, debugSymbolStream);
-                    }
-                }
-                else
-                {
-                    return AssemblyLoadContext.Default.LoadFromStream(assemblyStream);
-                }
-            }
-#elif NET46
+#if NETSTANDARD2_0
             return Assembly.Load(assembly, debugSymbols);
```

Comment:
This doesn't look right... now the .NET version of this will not work correctly, right?

Reply:
We're not multi targeting PlatformServices anymore so it is not an issue.